packages/nextjs/services/onchainImpactDashboardApi/projectService.ts

```diff
@@ -0,0 +1,21 @@
+import { Project } from "../database/schema";
+
+export const ProjectService = () => {
```

Comment:
Curious why you made this wrapper function, rather than just exporting `getProjectById()` and `getPaginatedProjects()` individually? Is this a service pattern you've used in the past?

Reply:
Yes, in this way with the destructuring on the consumer you define just the method, also is not needed here but with and state is a closure you can use in other situations.

Reply:
for example here you can apply memoize techniques storing data on a variable and returning only the functions